configure.ac

```diff
@@ -190,11 +191,13 @@ if [[ "$a8_host" = "falcon" ]]; then
 	CFLAGS="-m68020-60 -O2 -fomit-frame-pointer -Wall"
 	LDFLAGS="-m68020-60"
 	SUPPORTS_RDEVICE=no
+    SUPPORTS_NETSIO=no
 fi
 if [[ "$a8_host" = "firebee" ]]; then
 	CFLAGS="-mcpu=5475 -O2 -fomit-frame-pointer -Wall"
 	LDFLAGS="-mcpu=5475"
 	SUPPORTS_RDEVICE=no
+    SUPPORTS_NETSIO=no
```

Comment:
Indentation (perhaps just make the whole block aligned with spaces, not sure where the tabs come from).